Framework/DataHandling/src/ISISDataArchive.cpp

```diff
@@ -55,19 +54,22 @@ ISISDataArchive::getArchivePath(const std::set<std::string> &filenames,
     g_log.debug() << ext << ")\n";
```

Comment:
Can we wrap the above 2 loops with an `if (g_log.is(Logger::Priority::Debug)) {... }` so that the loops are not even executed if the log level is not debug?